Maddori.Apple/Maddori.Apple/Global/Extension/Rx/UIViewController+Rx.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  UIViewController+Rx.swift
+//  Maddori.Apple
+//
+//  Created by 이성민 on 11/9/23.
+//
+
+import UIKit
+
+import RxCocoa
+import RxSwift
+
+public extension Reactive where Base: UIViewController {
+    var viewDidLoad: Observable<Void> {
+        return self.methodInvoked(#selector(Base.viewDidLoad)).map { _ in }
+    }
```

Comment:
그렇다면 

이 부분을 

```suggestion
    var viewDidLoad: Observable<Void> {
        return Observable.just(())
    }
```

이렇게 바꾸면 어때용 ㅋㅋ

Reply:
[58a9f82](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/400/commits/58a9f828b32bbcdde6282c16161b9b89e525187c)

반영 완 !